api/v1alpha1/helmchartproxy_webhook.go

```diff
@@ -38,13 +44,31 @@ func (r *HelmChartProxy) SetupWebhookWithManager(mgr ctrl.Manager) error {
 
 var _ webhook.Defaulter = &HelmChartProxy{}
 
+var trueValuePtr = pointer.Bool(true)
```

Comment:
Nit: I think the convention we've been using has just been to use the function return value instead of declaring a variable for it

Reply:
sure, updated. Please review.